cmd/geth/main.go

```diff
@@ -155,6 +155,7 @@ var (
 		utils.MinerNotifyFullFlag,
 		utils.IgnoreLegacyReceiptsFlag,
 		utils.RollupSequencerHTTPFlag,
+		utils.RollupHistoricalHttpFlag,
```

Comment:
Can we remove "http" from the name? Maybe just name it RPC?
The same holds for the old `RollupSequencerHTTPFlag` which slipped through; websockets and IPC are valid too, it's not always http.

Reply:
Addressed here: https://github.com/ethereum-optimism/op-geth/pull/11 (In draft while CI is running).